### PR TITLE
Documentation #11

### PR DESCRIPTION
**Pull request by Himasnhu-AT**
#11

### CODE REVIEW

**Review by techsavvyash — APPROVED** [review]
👍🏼

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+
+```bash
+git clone https://github.com/techsavvyash/dynamo-prisma.git
+npm link
+```
+
+### using npm
+
+```bash
+npm install @soorajk1/dynamo-prisma
```
Comment (sooraj1002) [inline README.md:17]: this should be @techsavvyash/dynamo-prisma as per the latest package.json

**Comment by sooraj1002** [comment]
@Himasnhu-AT could comment out the rest of `schemaGenerator.spec` as part of the PR? it causes the build to fail

File: docs/functions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Follow guide given in [README.md](../README.md)
+
+- #### generateSchemaFromJson(jsonData, prismaFilePath);
+
+  Argument of function:
+
+  - `jsonData: Schema` - JSON as per schema model which is defined in [JSON Schema](./schema.md)
+
+  - `prismaFilePath: string` - Path of the prisma schema file. If file is present, it will append the new schema to the existing schema. If file is not present, it will create a new schema file.
```
Comment (sooraj1002) [inline docs/functions.md:13]: mark `prismaFilePath`  as an optional field. update the default location which the package will look for

File: docs/functions.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- #### GenerateSchemaFile(filePath);
+
+  Argument of function:
+
+  - `filePath: String` - file path of the json file.
+
+  Returns: {status: boolean, message: string, error: string}
+
+- #### JsonChecks;
```
Comment (sooraj1002) [inline docs/functions.md:25]: This function is not currently exported by the package

**Review by sooraj1002 — CHANGES_REQUESTED** [review]

File: docs/functions.md (hunk 3 above)
Comment (Himasnhu-AT) [reply docs/functions.md:25]: okk, i'll remove it and update index.ts

File: docs/functions.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ Follow guide given in [README.md](../README.md)
 
   - `jsonData: Schema` - JSON as per schema model which is defined in [JSON Schema](./schema.md)
 
-  - `prismaFilePath: string` - Path of the prisma schema file. If file is present, it will append the new schema to the existing schema. If file is not present, it will create a new schema file.
+  - OPTIONAL `prismaFilePath?: string` - Path of the prisma schema file. If file is present, it will append the new schema to the existing schema. If file is not present, it will create a new schema file.
```
Comment (sooraj1002) [inline docs/functions.md:13]: add what the default would be as well.

**Pull request merged**